Sample/Sample/AppDelegate.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AppDelegate.swift
+//  Sample
+//
+//  Created by Denis on 22/10/2018.
```

Comment:
Do we really need it? Also, how is the date formatted?
Picture-reminder 😄 
![image](https://user-images.githubusercontent.com/1836384/47439747-9691f380-d7b5-11e8-994b-1b2347034639.png)